client/state/sites/reducer.js

```diff
@@ -75,6 +75,15 @@ export function items( state = {}, action ) {
 			}
 			return state;
 
+		case 'PRESSABLE_TEST':
+			const originalSite = state[ action.siteId ];
+			if ( originalSite ) {
+				return Object.assign( {}, state, {
+					[ action.siteId ]: merge( {}, originalSite, { jetpack: true, options: { pressable: true } } )
```

Comment:
This is a good start, but we'll probably also want to mark when a site is in the process of transferring.